README.md

```diff
@@ -7,6 +7,9 @@
 [Development guide](docs/GET_STARTED.md)
 
 ## Release notes
+[15/05/2020] 1.0.1
```

Comment:
1.0.2?
As what I learnt from last release, this README.md may not reflect on npm. The actual place may need to change is `cennznet/types/README.md`?

Reply:
I feel we might not want to put release notes in README at all..they should not be part of read me

Reply:
Yeah, agreed. It is better in `CHANGELOG.md`. As the time we refactored on API, we did not have considered a proper CHANGELOG at all, so decided to put a simple release notes at the top of README.

Could consider move to a changelog, and make a link to readme, or anyway consumers can easily find this kind of release notes..